woocommerce/woocommerce
# [Accessibility] Add scope attributes to the order table on My Account

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:




This PR adds the `scope` HTML attribute to improve the accessibility of the orders table on the My Account page.
Closes #43629 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

This PR adds more semantics to the Orders table.
The following steps to QA consider the test with VoiceOver.

1. Create some orders.
2. On the store, go to the My Account -> Orders.
3. Turn on the screen reader.
4. Navigate through the table cells. On VoiceOver, pressing "Control + Option + Arrow keys" makes this possible.
5. When you jump to a new row for the first time, the content of the first cell in the row should be announced. This is because the first cell has been turned into the header of the row.
6. On VoiceOver, When you jump to the first row, where all the headers leave, the content of the first cell in the second row is also announced. This is due to the change of the HTML tag of this cell from `<td>` to `<th>`. The same behavior can be found on the W3C example of an accessible table: https://www.w3.org/WAI/tutorials/tables/two-headers/ 




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
